changelog/1.33.3.md

```diff
@@ -0,0 +1,26 @@
+---
+title: "1.33.3"
+description: "Released on 08/22/2022"
+---
+
+### Breaking changes ❗
+
+There are no breaking changes in 1.33.3.
+
+### Features ✨
+
+- infra: allow overriding BitBucket OAuth consumer key using the
+  `CODERD_BITBUCKET_CONSUMER_KEY` environment variable.
+- cli: add `--duration` flag to `coder tokens create` to control token lifetime.
+
+### Bug fixes 🐛
+
+- infra: fix workspace builds being stuck on "enqueuing workspace build" step
+  due to nil pointer panic. Workspaces that were getting stuck should now show
+  a proper root cause error in the build log.
+- infra: reduce log spam in coder agent log file in workspaces.
+- infra: upgrades code-server to 4.6.0 to fix frequent disconnects.
```

Comment:
I am not sure if this is too specific to include, but mentioning the type of disconnects that were fixed could make sense in case there are still other disconnect bugs lying around somewhere.

```suggestion
- infra: upgrades code-server to 4.6.0 to fix frequent disconnects caused by reverse proxy idle timeouts.
```